src/components/workshops/CarouselProject.js

```diff
@@ -0,0 +1,198 @@
+import React, { Component, Fragment } from 'react'
+import api from 'api'
+import styled from 'styled-components'
+import { isURL } from 'validator'
+import {
+  Box,
+  Flex,
+  Label,
+  Heading,
+  Link as A,
+  Text,
+  Image,
+  Button,
+  theme,
+} from '@hackclub/design-system'
+
+const ProjectOuter = styled(Flex).attrs({
+  p: [0, 0, 4],
+  bg: [theme.colors.white, theme.colors.white, theme.colors.white],
```

Comment:
Background color (or regular color) can't be set responsively/as an array.
```suggestion
  bg: 'white',
```